source/gui/frmhelpgeneratorunit.pas

```diff
@@ -1270,6 +1273,7 @@ procedure TfrmHelpGenerator.btnOpenClick(Sender: TObject);
       CheckAutoLink.Checked := Ini.ReadBool('Main', 'AutoLink', false);
       CheckHandleMacros.Checked := Ini.ReadBool('Main', 'HandleMacros', true);
       CheckUseTipueSearch.Checked := Ini.ReadBool('Main', 'UseTipueSearch', false);
+    CheckOpenHTML.Checked := Ini.ReadBool('Main', 'OpenHTML', true);;
       rgLineBreakQuality.ItemIndex := Ini.ReadInteger('Main', 'LineBreakQuality', 0);
```

Comment:
This line should be indented the same as lines above. Also, at the there's double semicolon `;;` :)

Reply:
Fixed by rebasing and rewriting because of merge conflict